src/n2n.c

```diff
@@ -61,6 +61,54 @@ SOCKET open_socket (int local_port, in_addr_t address, int type /* 0 = UDP, TCP
 }
 
 
+int detect_local_ip_address (n2n_sock_t* out_sock, const n2n_edge_t* eee) {
```

Comment:
`n2n.c` is a file shared by edge and supernode, so there should be no `n2n_edge_t* eee`.

Reply:
You are right, will move it to `edge_utils.c`.